ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ExceptionLoggingWriteHandler.java

```diff
@@ -72,9 +75,33 @@ void updateOrCreateDefaultExceptionLoggingEnabledService(final OperationContext
             value.set(enabled);
         } else {
             // create and install the service
-            final ValueService<AtomicBoolean> service = new ValueService<>(new ImmediateValue<>(new AtomicBoolean(enabled)));
-            context.getServiceTarget().addService(serviceName, service)
-                    .install();
+            final ServiceBuilder<?> sb = context.getServiceTarget().addService(serviceName);
+            sb.setInstance(new ValueService(() -> new AtomicBoolean(enabled))).install();
```

Comment:
Similar to the above, we should create the AtomicBoolean eagerly and use a Service that provides a fixed value.

Reply:
Fixed